components/PageNavigation.vue

```diff
@@ -20,6 +20,31 @@
         </span>
       </SmartLink>
     </li>
+    <!-- sso links -->
+    <template v-if="enableAuthLinks">
+      <li
+        v-if="!isAuthenticated"
+        class="nav-item"
+      >
+        <b-link
```

Comment:
This does not render a routable link, which it should, to offer the option of logging in in a new tab should the user prefer.

Create a new minimal-layout login page which simply triggers `this.$auth.loginWith('keycloak');` in its `fetch` method, and have this `b-link` link to it via `:to`.

Reply:
Tried suggestion with ```fetch``` in minimal-layout login page and it's not working if login link is opened on new tab. It works if ```this.$auth.loginWith ``` is added to ```mounted```.